### ISSUE TITLE
could not start electron Error: spawn electron ENOENT

### ISSUE THREAD

**Issue by jemspatel** [brave/browser-laptop#3323]
**Did you search for similar issues before submitting this one?**
yes
**Describe the issue you encountered:**
when i execute command `npm install` then following error occur

```
LINK : fatal error LNK1181: cannot open input file 'C:\OpenSSL-Win64\lib\libeay32.lib' [E:\browser-laptop\node_modul
es\secp256k1\build\secp256k1.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (E:\browser-laptop\node_modules\npm\node_modules\node-gyp\lib\build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)
gyp ERR! System Windows_NT 10.0.10240
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "E:\\browser-laptop\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd E:\browser-laptop\node_modules\secp256k1
gyp ERR! node -v v6.2.0
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "E:\\browser-laptop\\node_modules\\npm\\bin\\npm-cli.js" "run" "rebuild"
npm ERR! node v6.2.0
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! secp256k1@3.0.1 rebuild: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the secp256k1@3.0.1 rebuild script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the secp256k1 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs secp256k1
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls secp256k1
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     E:\browser-laptop\node_modules\secp256k1\npm-debug.log
npm WARN install:secp256k1@3.0.1 secp256k1@3.0.1 install: `npm run rebuild`
npm WARN install:secp256k1@3.0.1 Exit status 1
```

execute `npm run watch` no any errror occur but
when execute `npm start` then 

```
> brave@0.11.6 start E:\browser-laptop
> node ./tools/start.js --debug=5858 --enable-logging --v=0 --enable-extension-activity-logging --enable-sandbox-logging --enable-dcheck

could not start electron Error: spawn electron ENOENT
```

**Expected behavior:**

- Platform (Win7, 8, 10? macOS? Linux distro?):
win10
- Brave Version:

- Steps to reproduce:
    1. npm install
    2. npm run watch
    3. npm start

- Screenshot if needed:

- Any related issues:

**Comment by bsclifton** [comment]
The 2nd part of your issue is currently known and tracked with https://github.com/brave/browser-laptop/issues/3142

The first one is interesting. @jemspatel looks like you have Visual Studio 2015... what service pack (update 3?) I did run into issues compiling electron w/ pre-update 2 VS2015

**Comment by bsclifton** [comment]
@jemspatel this issue should be fixed now, with d3c04c1#diff-3c56a79867d9b41d5950b61c8523f078

Can you get latest from master and try again? Please let us know if it works. Thanks! :smile:
